src/pretix/control/navigation.py

```diff
@@ -518,3 +480,65 @@ def merge_in(nav, newnav):
                 parents[0]['children'].append(item)
                 continue
         nav.append(item)
+
+
+def get_admin_navigation(request):
+    url = request.resolver_match
+    if not url:
+        return []
+    nav = [
+        {
+            'label': _('Dashboard'),
+            'url': reverse('control:admin.dashboard'),
+            'active': 'dashboard' in url.url_name,
+            'icon': 'dashboard',
+        },
+        {
+            'label': _('Organizers'),
+            'url': reverse('control:admin.organizers'),
+            'active': 'organizers' in url.url_name,
+            'icon': 'group',
+        },
+        {
+            'label': _('Users'),
+            'url': reverse('control:admin.users'),
+            'active': False,
+            'icon': 'user',
+            'children': [
+                {
+                    'label': _('All users'),
+                    'url': reverse('control:admin.users'),
+                    'active': ('users' in url.url_name),
+                },
+                {
+                    'label': _('Admin sessions'),
+                    'url': reverse('control:admin.user.sudo.list'),
+                    'active': ('sudo' in url.url_name),
+                },
+            ]
+        },
+        {
+            'label': _('Global settings'),
+            'url': reverse('control:admin.global.settings'),
+            'active': False,
+            'icon': 'wrench',
+            'children': [
+                {
+                    'label': _('Settings'),
+                    'url': reverse('control:admin.global.settings'),
+                    'active': (url.url_name == 'admin.global.settings'),
+                },
+                {
+                    'label': _('Update check'),
+                    'url': reverse('control:admin.global.update'),
+                    'active': (url.url_name == 'admin.global.update'),
+                },
+            ]
+        },
+    ]
+
+    merge_in(nav, sorted(
+        sum((list(a[1]) for a in nav_global.send(request, request=request)), []),
```

Comment:
Is this code tested?
`sorted` expects an iterator/sequence to be passed, but you pass a number (`sum` returns number).

Reply:
Hi @hongquan, this has been tested and no issue was found.
in this case, nav_global.send(request, request=request) is a tuple, and a[1] is a dict, so sum will return a dict list.